osdsctl/cli/volume.go

```diff
@@ -76,7 +76,41 @@ var (
 	volAz     string
 )
 
+var (
+	volLimit     string
+	volOffset    string
+	volSortDir   string
+	volSortKey   string
+	volId        string
+	volCreatedAt string
+	volUpdatedAt string
+	volTenantId  string
+	volUserId    string
+	volSize      string
```

Comment:
why not the type of volSize is string rather than int64.

Reply:
String in cobra is more better handling.